docs/source/top_level_elements/name.rst

```diff
@@ -341,6 +343,35 @@ An example is `insurancea:163 <https://digital.lib.utk.edu/collections/islandora
 
 We don't need to keep this.
 
+Names Missing Roleterms
+-----------------------
+
+We have a small number of names that have no roles.  Because the role will be used to express property, it is critical that
```

Comment:
I'm confused by this one. These all have a **roleTerm** of  **Associated name** with a URI of http://id.loc.gov/vocabulary/relators/asn. I know you point out above that Associated is not from marc relators, but Associated name seems to be acceptable.

```
<name authority="naf" type="corporate" valueURI="">
  <namePart>Bemis Bro. Bag Company</namePart>
    <role>
      <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/asn">Associated name</roleTerm>
  </role>
</name>
```

Reply:
We cleared this up during our conversation on 10/6. My confusion here was conflating two different ideas and thinking the issue here with these was a roleTerm, when it was the missing the valueURI of name.